recipe/meta.yaml

```diff
@@ -27,14 +27,17 @@ requirements:
         - zlib
 
 test:
+    requires:
+        - python {{ environ['PY_VER'] + '*' }}  # [win]
```

Comment:
All the automated scripts know how to work with this now so it won't introduce any problems on that side.